bundles/AdminBundle/Resources/public/js/pimcore/document/editables/image.js

```diff
@@ -264,6 +273,7 @@ pimcore.document.editables.image = Class.create(pimcore.document.editable, {
             this.datax.id = data.id;
 
             this.updateImage();
+            this.checkValue();
```

Comment:
```suggestion
            this.checkValue(true);
```
image editable should reset error border on dragndrop